src/Spectre.AlgorithmsCli/Spectre.AlgorithmsCli.vcxproj

```diff
@@ -60,9 +60,10 @@
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <AdditionalOptions>/std:c++latest %(AdditionalOptions)</AdditionalOptions>
       <TreatWarningAsError>true</TreatWarningAsError>
+      <AdditionalIncludeDirectories>$(SolutionDir)Spectre.libDivik;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Actually, deleting this causes some bizarre problems. The files are seen from the IDE but the project does not compile because of "not finding the file/directory".